item.rb

```diff
@@ -21,18 +24,13 @@ def add_label(label)
     @label = label
   end
 
-  def move_to_archive
-    @archived = true if can_be_archived?
-  end
-
-  private
-
-  attr_reader :id
-  attr_accessor :archived
-
   def can_be_archived?
     return true if DateTime.now.year - @publish_date.year > 10
 
     false
   end
```

Comment:
![image](https://user-images.githubusercontent.com/42799579/188539860-c230dee7-75de-4037-b3cc-a9ee0a2abc41.png)
The method #can_be_archived? inside 'item.rb' needs to be private, as specified on the UML class diagram, as well as the one who inherits from it in MusicAlbum.

Reply:
I didn't touch this part of code

Reply:
> 2. The method #can_be_archived? inside 'item.rb' needs to be private, as specified on the UML class diagram below.
>    ![image](https://user-images.githubusercontent.com/42799579/188539860-c230dee7-75de-4037-b3cc-a9ee0a2abc41.png)
>    As well as the MusicAlbum's:
>    ![image](https://user-images.githubusercontent.com/42799579/188540260-57d23e66-6321-480f-b3b3-4d60b87658f3.png)

Please add the keyword 'private' and put all private methods below it. It should look like:

private

def can_be_archived?
   ... [logic] ...
end